Dockerfile

```diff
@@ -51,5 +51,7 @@ RUN ./bin/insolar bootstrap --config scripts/insolard/bootstrap.yaml \
 RUN go get github.com/fullstorydev/grpcurl
 RUN go install github.com/fullstorydev/grpcurl/cmd/grpcurl
 
+RUN sed -i 's;eu-west-1.ec2.archive.ubuntu.com;eu.archive.ubuntu.com;' /etc/apt/sources.list
```

Comment:
I think it will be better to add multiple alternative repositories to a *.list file in /etc/apt/sources.list.d, so that, if the first repository is unavailable, apt-get will try to download the same package from the next repository in the list. 
WDYT? 
And you have to resynchronize the package index files from the sources with the command `apt-get update `